### PR TITLE
Add build step

### PR DESCRIPTION
**Pull request by anxolin**
# Description
I get this error if i try to follow the README:

image

# Changes
Added further instructions in README

### CODE REVIEW

**Comment by anxolin** [comment]
Mmm, not even with the build it worked for me 

image

**Comment by anxolin** [comment]
A clean clone made it work
image

File: README.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ The documentation is built using [Docusaurus 2](https://docusaurus.io/), a moder
 yarn
 ```
 
+You will also need to `build` the app, to ensure external dependent projects are cloned and setup properly.
+
+```bas
```
Comment (mfw78) [inline README.md]:
```suggestion
```bash
```

**Review by mfw78 — APPROVED** [review]
typo

**Pull request merged**